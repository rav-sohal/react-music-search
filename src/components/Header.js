import React from 'react';
import {Navbar} from 'react-bootstrap';

const logoBrand = {
    fontSize: '38px',
    fontWeight: 'bolder'
}

const Header = () => (
  <Navbar>
    <div className="container">
      <Navbar.Brand href="#home" style={logoBrand}>SYFM</Navbar.Brand>
      <Navbar.Toggle />
      <Navbar.Collapse className="justify-content-end">
        <Navbar.Text>
          Search your favourite music
        </Navbar.Text>
      </Navbar.Collapse>
    </div>
  </Navbar>
);

export default Header;
