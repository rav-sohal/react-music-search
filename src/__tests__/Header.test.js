import {render, screen} from '@testing-library/react';
import {create} from 'react-test-renderer';
import Header from '../components/Header';

describe ('Header.js tests', () => {
  test ('Snapshot test of Header', () => {
    const testHeaderSnapshot = create (<Header />).toJSON ();
    expect (testHeaderSnapshot).toMatchSnapshot ();
  });
});