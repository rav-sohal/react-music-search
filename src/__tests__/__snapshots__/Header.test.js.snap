// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.js tests Snapshot test of Header 1`] = `
<nav
  className="navbar navbar-expand navbar-light"
>
  <div
    className="container"
  >
    <a
      className="navbar-brand"
      href="#home"
      style={
        Object {
          "fontSize": "38px",
          "fontWeight": "bolder",
        }
      }
    >
      SYFM
    </a>
    <button
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="justify-content-end navbar-collapse collapse"
    >
      <span
        className="navbar-text"
      >
        Search your favourite music
      </span>
    </div>
  </div>
</nav>
`;
